### DISCUSSION TITLE
大家可以在这个讨论下，探讨你对这个项目的想法

### DISCUSSION THREAD

**Discussion by BaoXuebin** [BaoXuebin/beancount-gs#69] [Q&A]
一起参与共建吧 :-)

**Comment by linllyw** [comment]
能支持股票交易和资产交易吗？比如房产之类的

**Reply by BaoXuebin** [reply]
https://beancount.github.io/docs/how_inventories_work
https://www.yuque.com/chuyi-ble7p/beancount-gs/klu38y

你可以参考下这些文档，希望对你有帮助。
